### PR TITLE
Fix selector to follow hierarchy

### PR DESCRIPTION
**Pull request by alexandref75**
Signed-off-by: Alexandre Peixoto Ferreira <alexandref75@gmail.com>

### CODE REVIEW

**Comment by ericvh** [comment]
sorry, this came in during refactoring, can you reapply on the the new CI configuration?

**Comment by alexandref75** [comment]
Rebased.

File: .github/workflows/docker-buildx.yml [hunk 1]
```diff
@@ -72,14 +72,14 @@ jobs:
       - name: Setup Docker buildx
         uses: docker/setup-buildx-action@v2.2.1
         with:
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
```
Comment (ericvh) [inline .github/workflows/docker-buildx.yml]: leave linux/arm/v7 out - it causes build issues.

**Comment by ericvh** [comment]
we'll need to bump chart version

**Review by jishminor — APPROVED** [review]
LGTM

**Pull request merged**